torch/csrc/jit/passes/freeze_module.cpp

```diff
@@ -332,12 +395,32 @@ class AttributePropagator {
               auto attr = module.attr(name);
               insertMutableAttr(name, attr, mptr);
               if (attr.isModule()) {
-                modules.insert(attr.toModule()._ivalue());
+                auto module = attr.toModule();
```

Comment:
why is this change needed? if we insert the module and it already exists that should be fine